src/lib/certificate-manager.ts

```diff
@@ -316,8 +319,9 @@ export async function generateSiteCertificate(
  */
 export function deleteSiteCertificate( domain: string ): boolean {
 	try {
-		const siteCertPath = path.join( CERT_DIRECTORY, 'domains', `${ domain }.crt` );
-		const siteKeyPath = path.join( CERT_DIRECTORY, 'domains', `${ domain }.key` );
+		const punycodeDomain = domainToASCII( domain );
+		const siteCertPath = path.join( CERT_DIRECTORY, 'domains', `${ punycodeDomain }.crt` );
+		const siteKeyPath = path.join( CERT_DIRECTORY, 'domains', `${ punycodeDomain }.key` );
```

Comment:
```suggestion
		const siteCertPath = path.join( CERT_DIRECTORY, 'domains', `${ domain }.crt` );
		const siteKeyPath = path.join( CERT_DIRECTORY, 'domains', `${ domain }.key` );
```
Same thing here about the path not needing to be punycode-encoded.

Reply:
TIL. thanks for suggestion. I have removed it.